docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,572 @@
+
+## How to target .NET Core
+
+For the purposes of building a library, targeting ".NET Core" means targeting the .NET Standard Platform.  To see what that means exactly, see [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md).
+
+A distilled version is this:
+
+You need to pick a version of `netstandardX.X` (where `X.X` is a version number) to add to your `project.json` file.
+
+1. If you don't care about backwards compatibility, target `netstandard1.4`.
+2. If you care about compatibility with any of the following:
+
+    - .NET Framework 4.6.1, 4.6, 4.5.2, 4.5.1, or 4.5
+    - Windows Phone
+    - Windows Phone Silverlight
+    - Universal Windows Platform
+    - DNX Core
+    - Xamarin Platforms
+    - Mono 
+
+    Then [refer to this table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#mapping-the-net-platform-standard-to-platforms) and choose the version of the `dotnetXX` moniker you need.
+    
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `netstandard1.4`:
+
+    ```javascript
+    {
+        "frameworks":{
+            "netstandard1.4":{}
+        }
+    }
+    ```
+    
+    If you wanted to be compatible with Windows Phone Silverlight 8, you would pick `netstandard1.1`:
+    
+    ```javascript
+    {
+        "frameworks":{
+            "netstandard1.1":{}
+        }
+    }
+    ```
+3. If you care about compatibility with the .NET Framework versions 4.0 or below, or need to use .NET Framework APIs not available in .NET Core, you'll have to read the next sections.
+
+## How to target both .NET Core and Older .NET Framework Versions
+
+**NOTE:** This requires Windows and .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+
+Say you wish to support older frameworks, such as .NET Framework 4.5.2.  From a project targeting perspective, this is quite simple!  You will need to specifically target the version of .NET Framework with its relevant Target Framework Monikers.  These are mapped out here:
+
+```
+.NET Framework 2.0   --> net20
+.NET Framework 3.0   --> net30
+.NET Framework 3.5   --> net35
+.NET Framework 4.0   --> net40
+.NET Framework 4.5   --> net45
+.NET Framework 4.5.1 --> net451
+.NET Framework 4.5.2 --> net452
+```
+
+For example, you may need to support .NET Framework 4.0 and .NET Framework 4.5.2, but wish to use some of the newer APIs available in .NET Core and .NET Framework 4.6.  Setting up your `project.json` file like this will do the trick:
```

Comment:
I think of that as multi-targeting.  But I can see how cross-compilation could mean the same thing.  Can't think of a good name for when you want to use `#if`.